java-generator/core/src/main/java/io/fabric8/java/generator/CRGeneratorRunner.java

```diff
@@ -28,13 +28,16 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
 import java.util.Objects;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
 public class CRGeneratorRunner {
 
   private final Config config;
+  private static final List<String> STD_PROPS = Arrays.asList("metadata", "spec", "status", "apiVersion", "kind");
```

Comment:
Shouldn't this be: `"metadata", "apiVersion", "kind"` ?

Reply:
`spec` and `status` are already params in the constructor of `JCRObject` hence they are ignored as top level fields